cfgov/legacy/static/nemo/_/c/v1-nonresponsive-header.css

```diff
@@ -1069,6 +1069,7 @@ p+.m-global-banner_head,ul+.m-global-banner_head,ol+.m-global-banner_head,dl+.m-
     letter-spacing: 1px;
     line-height: 1.25;
     text-transform: uppercase;
+    padding-left: 1.42857143em;
```

Comment:
Please add a note about this is necessary.

Reply:
Added in https://github.com/cfpb/cfgov-refresh/pull/3280/commits/7f0fedc0e7419fcf8505398b8a4499afffce6a61